New-version/pagos-sin-cvv (#2291)
mercadopago/devsite-docs: guides/payments-without-cvv/prerequisites.es.md

# Requisitos previos

Para integrar la solución de pagos sin la necesidad de utilizar el CVV (Card Verification Value) de la tarjeta para pagos recurrentes, es necesario cumplir con los requisitos enumerados a continuación.

| Requisitos | Descripción |
| --- | --- |

> | Contacto comercial | Para utilizar la función de pagos automáticos, es necesario ponerse en contacto con su representante comercial en Mercado Pago, quien proporcionará toda la información necesaria para completar la integración. |

Reviewer: ```suggestion
| Contacto comercial | Para utilizar la función de pagos automáticos, es necesario ponerse en contacto con el representante comercial de Mercado Pago, quien proporcionará toda la información necesaria para completar la integración. |
```

> | Cuenta de vendedor de Mercado Pago o Mercado Libre | Para utilizar la función de pagos automáticos, necesitas una cuenta de vendedor en Mercado Pago o Mercado Libre. Sino, [haz clic aquí](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing) para crearla gratis. | 

Reviewer: ```suggestion
| Cuenta de vendedor de Mercado Pago o Mercado Libre | Para utilizar la función de pagos automáticos, necesitas una cuenta de vendedor en Mercado Pago o Mercado Libre. Si todavía no tienes una, [haz clic aquí](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing) para crearla gratis. | 
```

> | Aplicación  | Las aplicaciones son las diversas integraciones contenidas en una o varias tiendas. Puedes crear una aplicación para cada solución que implementes a fin de tener todo organizado y mantener un control que facilite la gestión.Para operar pagos automáticos y sin CVV, la aplicación debe estar habilitada para este propósito. Estas autorizaciones son otorgadas por Mercado Pago, por lo que debes enviarnos el **ID de la aplicación** para realizar la configuración correspondiente. <br><br> Consulta tu [Tus integraciones](/developers/es/docs/checkout-bricks/additional-content/your-integrations/introduction) para obtener más información sobre cómo crear una aplicación. |

Reviewer: ```suggestion
| Aplicación  | Las aplicaciones son las diversas integraciones contenidas en una o varias tiendas. Puedes crear una aplicación para cada solución que implementes a fin de tener todo organizado y mantener un control que facilite la gestión. Para operar con pagos automáticos y sin CVV, la aplicación debe estar habilitada para este propósito por Mercado Pago, por lo que debes enviarnos el **ID de la aplicación** para realizar la configuración correspondiente. <br><br> Consulta tu [Tus integraciones](/developers/es/docs/checkout-bricks/additional-content/your-integrations/introduction) para obtener más información sobre cómo crear una aplicación. |
```
